pytorch_lightning/trainer/callback_hook.py

```diff
@@ -105,12 +105,12 @@ def on_test_epoch_end(self):
             callback.on_test_epoch_end(self, self.lightning_module)
 
     def on_epoch_start(self):
```

Comment:
how the body differs, shall we add an argument about these? or we just relied on Trainer phase train/val/test?